usautobuild/actions/gitter.py

```diff
@@ -57,3 +57,31 @@ def update_repo(self) -> None:
     def start_gitting(self) -> None:
         self.prepare_git_directory()
         self.config.project_path = self.local_repo_dir / "UnityProject"
+
+    def get_Good_file_tag(self) -> str:
+        log.debug("Searching for the latest 'good-file-*' tag...")
+
+        # Fetch all tags from the repository
+        tags = self.local_repo.tags
+
+        # Filter tags that start with 'good-file-'
+        good_file_tags = [tag for tag in tags if tag.name.startswith("good-file-")]
```

Comment:
what are the tags going to be called? This doesn't look like semantic versioning, it should just be ``v1.0.0``

Reply:
![image](https://github.com/user-attachments/assets/9bd68531-80fc-467f-8c69-4d10c8d065b8)
is how they look